common/check_fts_backlog

```diff
@@ -82,87 +74,81 @@ if __name__ == "__main__":
-                probe_metrics.gauge('fts3.{hostname}.submitted').labels(hostname=hostname).set(summary['submitted']
-                                                                                               + summary['active']
-                                                                                               + summary['staging']
-                                                                                               + summary['started'])
-
-                g.labels(**{'hostname': hostname}).set((summary['submitted'] + summary['active'] + summary['staging'] + summary['started']))
-                retvalue = OK
-                break
-            except Exception as error:
-                retvalue = CRITICAL
-                if result and result.status_code:
-                    errmsg = 'Error when trying to get info from %s : HTTP status code %s. [%s]' % (
-                        ftshost, str(result.status_code), str(error))
-                else:
-                    errmsg = 'Error when trying to get info from %s. %s' % (ftshost, str(error))
-        if retvalue == CRITICAL:
-            print("All attempts failed. %s" % errmsg)
-        WORST_RETVALUE = max(retvalue, WORST_RETVALUE)
-
-    if len(PROM_SERVERS):
-        for server in PROM_SERVERS:
-            try:
-                push_to_gateway(server.strip(), job='check_fts_backlog', registry=registry)
-            except:
-                continue
+
+                        if CHECK_BUSY and 'submitted' in channel and channel['submitted'] >= busylimit:
+                            url_activities = f'{ftshost}/fts3/ftsmon/config/activities/{VO}?source_se={src}&dest_se={dst}'
+                            activities = {}
+                            try:
+                                s = requests.get(url_activities, verify=False, cert=(PROXY, PROXY))
+                                for key, val in s.json().items():
+                                    activities[key] = val['SUBMITTED']
+                            except Exception as error:
+                                pass
+                            busy_channels.append({'src': src, 'dst': dst, 'submitted': channel['submitted'],
+                                                  'activities': activities})
+                    summary = res['summary']
+                    hostname = hostname.replace('.', '_')
+
+                    for state in ['submitted', 'active', 'staging', 'started']: 
+                        print(f'{hostname} : {state.capitalize()} : {summary[state]}')
+
+
+                    if busy_channels != []:
+                        print(f'Busy channels (>{busylimit} submitted):')
+                        for bc in busy_channels:
+                            activities_str = ", ".join([(f"{key}: {val}") for key, val in bc['activities'].items()])
+                            print(f'{bc['src']} to {bc['dst']} : {bc['submitted']} submitted jobs {activities_str}')
```

Comment:
This will not compile. Did you mean to use double quotes here? 
Also at other places for fstrings.

Reply:
I think a linter got a little excited and undid my work - thanks for catching it